test/batchReceiver.spec.ts

```diff
@@ -236,7 +236,7 @@ describe("ReceiveBatch from Queue/Subscription", function(): void {
     await testPeekMsgsLength(queueClient, 0);
 
     const deadLetterQueuePath = Namespace.getDeadLetterQueuePathForQueue(
-      process.env.QUEUE_NAME ? process.env.QUEUE_NAME : ""
+      queueClient.name ? queueClient.name : ""
```

Comment:
The `name` property is not optional, so you can skip the null check here. Same for topic/subscription client